### PR TITLE
feat: api support jwt

### PR DESCRIPTION
**Pull request by DDDHuang**

### CODE REVIEW

File: apps/emqx_dashboard/etc/emqx_dashboard.conf [hunk 1]
```diff
@@ -7,6 +7,8 @@ emqx_dashboard:{
     default_password: "public"
     ## notice: sample_interval should be divisible by 60.
     sample_interval: 10s
+    ## api jwt timeout. default is 30 minute
+    jwt_exptime: 30m
```
Comment (turtleDeng) [inline apps/emqx_dashboard/etc/emqx_dashboard.conf:11]: token_expired_time

File: apps/emqx_dashboard/src/emqx_dashboard_admin.erl [hunk 2]
```diff
@@ -170,35 +159,27 @@ check(Username, Password) ->
         [#mqtt_admin{password = <<Salt:4/binary, Hash/binary>>}] ->
             case Hash =:= md5_hash(Salt, Password) of
                 true  -> ok;
-                false -> {error, <<"Password Error">>}
+                false -> {error, <<"PASSWORD_ERROR">>}
```
Comment (tigercl) [inline apps/emqx_dashboard/src/emqx_dashboard_admin.erl:162]: Using `USERNAME_OR_PASSWORD_ERROR`，avoid exposing more information.

File: apps/emqx_dashboard/src/emqx_dashboard_admin.erl [hunk 3]
```diff
@@ -170,35 +159,27 @@ check(Username, Password) ->
...
             end;
         [] ->
-            {error, <<"Username Not Found">>}
+            {error, <<"USERNAME_ERROR">>}
```
Comment (tigercl) [inline apps/emqx_dashboard/src/emqx_dashboard_admin.erl:165]: Using `USERNAME_OR_PASSWORD_ERROR`，avoid exposing more information.

File: apps/emqx_dashboard/include/emqx_dashboard.hrl [hunk 4]
```diff
@@ -14,7 +14,18 @@
...
 
--record(mqtt_admin, {username, password, tags, role = undefined}).
+-record(mqtt_admin, {
+    username             :: binary(),
+    password             :: binary(),
+    tags                 :: list() | binary(),
+    role = undefined     :: atom()
+    }).
+
+-record(mqtt_admin_jwt, {
```
Comment (tigercl) [inline apps/emqx_dashboard/include/emqx_dashboard.hrl:24]: It should have better record name, `mqtt_admin` too. They have nothing to do with mqtt. You can improve this in the next version.

**Review by turtleDeng — APPROVED** [review]

**Pull request merged**